Makefile

```diff
@@ -26,9 +26,7 @@ SILENT ?= @
 # TODO: [ROX-19070] Update postgres store test generation to work for foreign keys
 UNIT_TEST_IGNORE := "stackrox/rox/sensor/tests|stackrox/rox/operator/tests|stackrox/rox/central/reports/config/store/postgres|stackrox/rox/central/auth/store/postgres|stackrox/rox/scanner/e2etests"
 
-ifeq ($(TAG),)
-TAG=$(shell git describe --tags --abbrev=10 --dirty --long --exclude '*-nightly-*')
-endif
+include make/tag.mk
```

Comment:
Not sure we are on the same page about the importance of this issue. 

I reverted the Makefile changes, as the image tag is produced correctly and the `go: command not found` stderr can only be seen in the log. 

So how do you suggest to address `GOGO_DIR` and `GRPC_GATEWAY_DIR`?
Sure we could add a check to see if `go` is installed, but won't this break other workflows?

Reply:
> Not sure we are on the same page about the importance of this issue.
I'm not sure our understanding of _what is_ the issue was the same and changed the same way throughout the life of this PR :wink:

I think, taking the debt of `go: command not found` lines in `stderr` is perhaps the least evil. Fixing GOGO_DIR and GRPC_GATEWAY_DIR alone won't be enough because SCANNER_DIR would still produce these errors.